app/web/features/auth/notifications/notificationUtils.ts

```diff
@@ -0,0 +1,16 @@
+const getCurrentSubscription = async () => {
+  let registration = await navigator.serviceWorker.getRegistration();
```

Comment:
Do you need to check if `serviceWorker` is in `navigator` here?

Reply:
Ah yeah that can't hurt. Just added.